ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.scss

```diff
@@ -50,12 +50,8 @@ section {
         box-sizing: border-box;
 
         .image {
-            flex: 0 1 400px;
+            width: 400px;
```

Comment:
Can you please fix styles for media queries (the layout is shifted)
![image](https://user-images.githubusercontent.com/7396837/122955936-3cdc8f80-d389-11eb-91f3-dd44eaf32c01.png)

![image](https://user-images.githubusercontent.com/7396837/122956026-52ea5000-d389-11eb-8012-22326cd40ca1.png)

Reply:
thanks! done

Reply:
suggest changing the `width: 400px` to 'width: 25rem' as to have the image fit the small (mobile) screens
(or change the width for mobile breakpoint)

Reply:
Fixed ^ at 5c57266989ab185c5506441ff913a49258ed0f1e